### ISSUE TITLE
Support for css border color as text and background

### ISSUE THREAD

**Issue by shaan1974** [jgthms/bulma#3533]
This is a feature request about **Bulma**.
This is about the Bulma **CSS framework**
I'm using Bulma **version 0.9.3**

### Description

Is it possible to add css "border-color" in **helpers** ->**color** as text color and background color.

**Comment by jrock17** [comment]
I agree. It would be nice to use `has-border-primary` the same way you can use `has-text-primary` and `has-background-primary` including the different shades.
